src/content/reference/react/useSyncExternalStore.md

```diff
@@ -57,6 +57,10 @@ The current snapshot of the store which you can use in your rendering logic.
 
 * If a different `subscribe` function is passed during a re-render, React will re-subscribe to the store using the newly passed `subscribe` function. You can prevent this by declaring `subscribe` outside the component.
 
+* If the store is mutated during a [non-blocking transition update](/reference/react/useTransition), React will fall back to performing that update as blocking. Specifically, React will call `getSnapshot` a second time just before applying changes to the DOM. If it returns a different value than when it was called originally, React will restart the transition update from scratch, this time applying it as a blocking update, to ensure that every component on screen is reflecting the same version of the store.
+
+* It's not recommended to _suspend_ a render based on a store value returned by `useSyncExternalStore`. For example, if you set `const expandedItemId = useSyncExternalStore(...)`, avoid rendering a [`lazy`](/reference/react/lazy) component conditionally like `{expandedItemId != null && <LazyItemDetails />}` or calling [`use`](/reference/react/use) with a Promise dependent on the value like `use(fetchItem(expandedItemId))`. The reason is that mutations to the external store cannot be [marked as non-blocking transition updates](/reference/react/useTransition), so they will trigger the nearest [`Suspense` fallback](/reference/react/Suspense), replacing already-rendered content on screen with a loading spinner, which typically makes a poor UX.
```

Comment:
> For example, if you set const expandedItemId = useSyncExternalStore(...), avoid rendering a lazy component conditionally like {expandedItemId != null && <LazyItemDetails />} or calling use with a Promise dependent on the value like use(fetchItem(expandedItemId)).

this sentence is a little hard to follow. Could you put the examples in to a code block? If I understand this correctly I think it should look something like this?:

```js
import { use, lazy } from 'react';

const MarkdownPreview = lazy(() => import('./MarkdownPreview.js'));

function TodosApp() {
  const todos = useSyncExternalStore(...);
  ❌ passing `todos` to the `use` Hook can suspend a render based on a store value
  use(fetchItem(todos[0]))
  ❌ conditionally rendering a lazy Component based on `todos` can suspend a render based on a store value
  return <>{todo != null && <MarkdownPreview />}</>
  // ...
}
```

Reply:
I wasn't sure if I'm allowed to put a code block inside a bullet point. I can play around with other treatments if you think this isn't parseable but this is an advanced concept (and advanced hook) so I figured it is ok if it isn't super simple.